RELEASE_NOTES.md

```diff
@@ -1,3 +1,22 @@
+### 1.0.4+167fae8 (Released 2024-1-15)
```

Comment:
Commits for 1.0.2 and 1.0.3 got lost (by me)? 

Did you increase patch version three times here?

Reply:
1.0.2, 1.0.3 and 1.0.4 are all part of this PR and are correctly written to RELEASE_NOTES.md